ssts/segmentation/segmentation_gmm.py

```diff
@@ -111,22 +111,8 @@ def transform(self, data, outliers=None):
             logger.warning("Attempting to transform prior to fitting. You must call .fit() first.")
             return None
 
-        if self.nonlinear:
-            data = SegmenterGMM.add_nonlinear_features(data)
-
         h, w, c = data.shape
-        n = h * w
-
-        if self.padding > 0:
-            data = SegmenterGMM.get_windows(data, padding=self.padding)
-        else:
-            data = data.reshape(n, c)
-
-        if self.normalize:
-            data = self.sst.transform(data) if self.zscale else SegmenterGMM.normalize_by_max(data)
-
-        if self.embedding_dim is not None:
-            data = self.pca.transform(data)
+        data = self.get_pca_components(data)
```

Comment:
@wesleyktatum I've created a method to extract pca_components seamlessly after fitting model